wordpress-mobile/gutenberg-mobile
# Enable sound on video block

Fixes wordpress-mobile/gutenberg-mobile#1072

gutenberg PR https://github.com/WordPress/gutenberg/pull/15997

To test:
- Test with both WPiOS and WPAndroid
- Verify that you hear sound from videos

(For iOS devices make sure that hardware switch to mute the device is not on)

Update release notes:
